.github/workflows/main.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macOS-latest]
-        dotnet: [5.0.100]
+        dotnet: [5.0.404]
```

Comment:
Needed to change it to fix https://github.com/dotnet/core/issues/6595#issuecomment-905086918